ParkingLot/src/main/java/services/ExitGate.java

```diff
@@ -1,28 +0,0 @@
-package services;
-
-import dao.TicketsDao;
-import factories.CostComputationFactory;
-import factories.ParkingSpotManagerFactory;
-import models.Ticket;
-
-public class ExitGate {
-    private final ParkingSpotManagerFactory parkingSpotManagerFactory;
-    private final CostComputationFactory costComputationFactory;
-    private final TicketsDao ticketsDao;
```

Comment:
Why is ExitGate entity having ticketsDao?

Reply:
To update end time in tickets.

Reply:
but it shouldn't right? ExitGate should be able to exist alone without ticketsDao.

Reply:
We'd like to maintain an internal record of what vehicles made exit at what time. This would be helpful to keep track of both in and out time of vehicles and also for cases if someone comes with a ticket that and asks for their vehicle after they already left the parking lot.